pilot/pkg/security/model/authentication.go

```diff
@@ -215,7 +213,7 @@ func ConstructSdsSecretConfig(name string, node *model.Proxy) *tls.SdsSecretConf
 				},
 			},
 			ResourceApiVersion:  core.ApiVersion_V3,
-			InitialFetchTimeout: features.InitialFetchTimeout,
+			InitialFetchTimeout: ptypes.DurationProto(time.Second * 0),
```

Comment:
This means a missing cert (which is possible for Gateways) can block envoy forever

Reply:
Ok. So we do want this SDS? If yes, I would  say we default to 15s here instead of forever because that is sensible default?. But current default is to block forever

Reply:
There is some thing odd about this though. When I made this change, tests are failing even though features.InitialFetchTimeout is same as what I changed here and sdsAdsConfig. Let me debug this bit more